tests/testthat/test-dataframe-s3-base.R

```diff
@@ -169,4 +172,10 @@ test_that("`[`'s drop argument works correctly", {
     test[1, character(), drop = TRUE],
     as_polars_df(NULL)
   )
+
+  # drop should be named
+  expect_equal(
+    test[, "a", TRUE],
+    pl$DataFrame(a = 1:3)
+  )
```

Comment:
Shouldn't we error here by checking empty dots?

Reply:
In general, I think the dots for extensions are silently ignored in S3 methods (because of the possibility of unknown arguments being passed for other classes).